src/WattleScript.Interpreter/Tree/Statements/ClassDefinitionStatement.cs

```diff
@@ -470,7 +470,7 @@ public override void Compile(FunctionBuilder bc)
                 bc.Emit_Literal(DynValue.NewString(field.Name));
             }
             if (privateCount > 0) {
-                bc.Emit_SetPriv(privateCount);
+                bc.Emit_SetFlags(privateCount, MemberModifierFlags.Private);
```

Comment:
Possible to complete with a list of public?

Reply:
fixed in 6f5e0fe